api.js

```diff
@@ -56,309 +60,310 @@ Api.prototype._runFile = function (file, runStatus, execArgv) {
 	});
 
 	var emitter = fork(file, options, execArgv);
-
 	runStatus.observeFork(emitter);
 
 	return emitter;
 };
 
+Api.prototype.run = function (files, options) {
+	var self = this;
+
+	var findOptions = {
+		cwd: this.options.resolveTestsFrom,
+		files: files
+	};
```

Comment:
The point of this is not the name (I named it `findOptions`, because `options` is already defined there), but extracting options to reduce nesting. I can move it back, not insisting on this one.